mobile/src/people/home-people-screen.tsx

```diff
@@ -18,10 +19,13 @@ const navBar =  new TopNavBar('People');
 
 class HomePeopleScreenImpl extends React.Component<PeopleScreenProps> {
     public static navigationOptions = navBar.configurate();
-
+    
     public render() {
+        
         return (
-            <Text>PeopleScreen. There are { this.props.departments.length} departments</Text>
+            <View style={{ flex: 1 }}>
+                <TopTabBarNavigator />
```

Comment:
Do you really need a component that only returns another one?

Reply:
are you referring to the whole file? I think yes, I'd rather have it as a wrapper. However, it should not have Props at all, they are not used anywhere.

Reply:
Text - forgot to delete it